src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,43 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String trackType;
+    private String cabType;
+    private int operatingWeight;
+
     public Bulldozer() {
     }
```

Comment:
delete unused constructors everywhere

Reply:
As I understand, I'm not allowed to do this, because no-arg constructors are used in the tests